services/src/main/java/org/exoplatform/news/queryBuilder/NewsQueryBuilder.java

```diff
@@ -54,7 +54,9 @@ public StringBuilder buildQuery(NewsFilter filter) throws Exception {
         }
         if (filter.getSearchText() != null && !filter.getSearchText().equals("")) {
           String escapedQuoteSearchText = filter.getSearchText().replace("'", "''").replace("\"", "\"\"");
-          sqlQuery.append("CONTAINS(.,'").append(escapedQuoteSearchText).append("')");
+          int index = escapedQuoteSearchText.indexOf("~0.6");
```

Comment:
The fuzzy index should never be used to extract the searched word as the index may be changed.

Reply:
@akhanfir my comment here is still relevent. where is this index added ? we should have the exact text without the fuy index